File: corpus/2018q1/_p2q1_ok/_check/A;0120.py
Scanner sc = new Scanner (System.in)
int tamanho,i,k
bool conf
print("insira o tamanho do vetor: ")
tamanho=int(input())
for i=0 in range(0, tamanho):
print("insira o numero que deseja conferir ")
k=int(input())
conf = comparaVetor (vetor,tamanho,k)
if conf == true:
    print ("Existe número igual a "+ k + "nesse vetor")
else:
def comparaVetor( int v[], int tamanho, int k):
    bool confere=true
    for i=0 in range(0, tamanho):
        confere = true
        else:
    return confere

File: corpus/2018q1/_p2q1_ok/_check/A;0120.java

import java.util.*;
public class Questao1 {

    public static void main(String[] args) {
        Scanner sc = new Scanner (System.in);
        int tamanho,i,k;
        boolean conf;

        System.out.print("insira o tamanho do vetor: ");
        tamanho=sc.nextInt();
        int vetor []=new int [tamanho];

        for (i=0;i<tamanho;i++){
            vetor[i]=sc.nextInt();
        }
        System.out.print("insira o numero que deseja conferir ");
        k=sc.nextInt();

         conf = comparaVetor (vetor,tamanho,k);

         if (conf == true){
             System.out.print ("Existe número igual a "+ k + "nesse vetor");
         }   
             else System.out.print ("Não existe número igual a" + k + "nesse vetor");

    }

    public static boolean comparaVetor ( int v[], int tamanho, int k){
        boolean confere=true;

        for (int i=0;i<tamanho;i++){
            if (v[i]==k){
                confere = true;
            }    
            else confere = false;    

    }
        return confere;
}
}
